coap_main.c

```diff
@@ -192,7 +192,7 @@ static CoAP_Result_t _rom SendResp(CoAP_Interaction_t* pIA, CoAP_InteractionStat
 
 		pIA->State = nextIAState; //move to next state
 
-		CoAP_EnqueueLastInteraction(pIA); //(re)enqueue interaction for further processing//todo: in die �u�ere statemachine
+		CoAP_EnqueueLastInteraction(pIA); //(re)enqueue interaction for further processing//todo: in die �u�ere statemachine
```

Comment:
I'm not sure what happened there. Files appear to be UTF-8, I ended up copying the original text over the mangled. and now appears the same, but Git thinks it's changed 🤷‍